applications/peeramid_labs.md

```diff
@@ -81,7 +87,7 @@
 
 **Similar Projects:**
 
-There are no similar projects in the Substrate/Polkadot/Kusama ecosystem. However, related ones in other ecosystems focus on decentralized governance and cryptographic voting systems. [Conviction voting](https://blog.giveth.io/conviction-voting-a-novel-continuous-decision-making-alternative-to-governance-aa746cfb9475) is a similar concept, which we elaborate to state of *conviction election.*
+There are no similar projects in the Substrate/Polkadot/Kusama ecosystem. However, related projects in other ecosystems focus on decentralized governance and cryptographic voting systems. Our research regarding [Conviction voting](https://blog.giveth.io/conviction-voting-a-novel-continuous-decision-making-alternative-to-governance-aa746cfb9475) discusses the different aspects of governance processes for implementation to Polkadot. We review the ability for the participants to repeatedly demonstrate their conviction to the same proposal with its changing alternatives and context. This approach may be used to prioritize proposals for voting when origin capacity to process them is limited, and to reflect the conviction of fellowship members.
```

Comment:
https://blog.giveth.io/conviction-voting-a-novel-continuous-decision-making-alternative-to-governance-aa746cfb9475 is not our research.

Reply:
it is "similar project"